en/app.ftl

```diff
@@ -46,6 +46,12 @@ nav-profile-sign-out-relay = Sign Out of { -brand-name-relay }
 nav-profile-sign-out-confirm = Are you sure you want to sign out?
 nav-profile-settings = Settings
 nav-profile-settings-tooltip = Configure { -brand-name-firefox-relay }
+nav-profile-help = Help and support
+nav-profile-help-tooltip = Get help using { -brand-name-relay }
```

Comment:
Is this available only for Premium users? If so, should we use a different brand placeholder?